pkg/adapter/config.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package adapter
+
+// EnvConfig is the minimal set of configuration parameters
+// source adapters must support
```

Comment:
Is this a "must support" or "a common set of useful environment variables"?

Reply:
The controllers provide these variables and the adapters *should* support them. Changed